build/Dockerfile

```diff
@@ -17,47 +17,48 @@ COPY . .
+    -o build/_output/bin/bootstrap \
+    cmd/bootstrap/main.go \
     && cp -r build/_output/bin/bootstrap /usr/local/bin/bootstrap
 RUN GOOS=$GOOS GOARCH=$GOARCH CGO_ENABLED=$CGO_ENABLED GO_LDFLAGS=$GO_LDFLAGS \
     go build -ldflags "-w -s -X main.GitCommit=$GIT_COMMIT -X main.GitBranch=$GIT_BRANCH -X main.BuildTime=$BUILD_TIME" \
-        -o build/_output/bin/healthcheck \
-            cmd/healthcheck/main.go \
+    -o build/_output/bin/healthcheck \
+    cmd/healthcheck/main.go \
     && cp -r build/_output/bin/healthcheck /usr/local/bin/healthcheck
 RUN GOOS=$GOOS GOARCH=$GOARCH CGO_ENABLED=$CGO_ENABLED GO_LDFLAGS=$GO_LDFLAGS \
     go build -ldflags "-w -s -X main.GitCommit=$GIT_COMMIT -X main.GitBranch=$GIT_BRANCH -X main.BuildTime=$BUILD_TIME" \
-        -o build/_output/bin/sidecar \
-            cmd/sidecar/main.go \
+    -o build/_output/bin/sidecar \
+    cmd/sidecar/main.go \
     && cp -r build/_output/bin/sidecar /usr/local/bin/sidecar
 RUN GOOS=$GOOS GOARCH=$GOARCH CGO_ENABLED=$CGO_ENABLED GO_LDFLAGS=$GO_LDFLAGS \
     go build -ldflags "-w -s -X main.GitCommit=$GIT_COMMIT -X main.GitBranch=$GIT_BRANCH -X main.BuildTime=$BUILD_TIME" \
-        -o build/_output/bin/peer-list \
-            cmd/peer-list/main.go \
+    -o build/_output/bin/peer-list \
+    cmd/peer-list/main.go \
     && cp -r build/_output/bin/peer-list /usr/local/bin/peer-list
 
 FROM redhat/ubi8-minimal AS ubi8
 
 # check repository package signature in secure way
 RUN export GNUPGHOME="$(mktemp -d)" \
-        && microdnf install -y findutils \
-        && gpg --keyserver keyserver.ubuntu.com --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
-        && gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
-        && rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona \
-        && curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
-        && rpmkeys --checksig /tmp/percona-release.rpm \
-        && rpm -i /tmp/percona-release.rpm \
-        && rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
-        && rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY \
-        && percona-release setup pdps-8.0
+    && microdnf install -y findutils \
+    && gpg --keyserver keyserver.ubuntu.com --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
+    && gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
+    && rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona \
+    && curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
+    && rpmkeys --checksig /tmp/percona-release.rpm \
+    && rpm -i /tmp/percona-release.rpm \
+    && rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
+    && rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY \
+    && percona-release setup pdps-8.0
 
 RUN set -ex; \
     microdnf update; \
+    microdnf install -y glibc-langpack-en; \
```

Comment:
you may consider doing it in a single `microdnf install`

Reply:
[16aab73](https://github.com/percona/percona-server-mysql-operator/pull/224/commits/16aab7318cb0c548e759f4368ce2bb6ad33dae12)